### PR TITLE
Milestone 1: HTML Structure for Youtube web-page

### PR DESCRIPTION
**Pull request by atoledo-ec**
# Basic HTML Structure based on Youtube
| Features | |
| --- | --- |
| Navbar | Like and Dislike buttons |
| Searchbar | List of recommended videos |
| Sidebar | Clickable logos and images |
| Playable video | Comment section |
| Description |

### CODE REVIEW

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey team,  your project is looking really good, however, to accomplish as the actual page, you required to do these changes, listed below:
 
_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, to get more knowledge and have a better project to add at your portfolio.
 
 
## [REQUIRED CHANGES]
#### Hey guys, your project looks amazing in terms of good practices (CSS, Semantic HTML, and so on), but you guys can improve the layout because currently, your page isn't looking like a clone.
- [x] Improve the recommended videos section.
- [x] Improve the video description section.
image
### Take those images as example:
- Yours
image
- Original
image


## [OPTIONAL CHANGES]
- [x] Implement the comments section


## Please make the required changes and resubmit for a new code review once you are ready💪

It was a pleasure to review your code! 


## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes required :recycle: 

### Hello team :wave: , you've done an amazing job on this project so far :muscle: :clap: , you just have to implement the finishing touches to get it approved. Kindly fix the required changes listed below:

**[ REQUIRED CHANGES]**

- [x] Update your readme file
- The getting started section needs to be changed. You have to modify that content to suit the context of your project.
- Check carefully for any other necessary adjustment(s)
- Add a live demo link of your project to your readme file. [This website](
https://raw.githack.com/) might help

**[GENERAL]**

- [x] In your index.html, the `<body>` tag is missing. 
- This is semantically wrong. Try to understand Semantic HTML from [here](https://www.w3schools.com/html/html5_semantic_elements.asp)


**[OPTIONAL]**

- [ ] Try to implement the footer section of the page.

* * *
### Kindly make these changes and request for another review.

**Best Regards,
Taofeek Olalere
GitHub: @teekaytech
Slack: @taofeekolalere**

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+<!--Main video section-->
+<section class="main-section">
+    <section class="main-video">
+        <div class="vid-frame">
+            <iframe class="video1" width="1031" height="580" src="https://www.youtube.com/embed/lJIrF4YjHfQ" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="vid-title">
+        <div class="prof-panel">
+                <h2 style="font-size: 25px; margin-bottom: 5px;">Embed Videos and playlists</h2>
```
Comment (umairarshadbutt) [inline index.html]: Remove inline style and manage it in external style sheet

File: index.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+            <div class="bot-prof-panel">
+                <div style="display: flex; width: 90px;">
```
Comment (umairarshadbutt) [inline index.html]: Remove inline style and manage it in the external style sheet

File: index.html [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        </div>
+        </div>
+        <div class="bottom-section">
+            <div class="profile-sec">
+                <div class="profile">
+                    <img src="https://yt3.ggpht.com/a/AATXAJwyEv0uEyk3JLskjLiUCqSVzsGmpWw2icLfBOBzWg=s48-c-k-c0xffffffff-no-rj-mo" alt="video profile pic">
+                </div>
+                    <div class="profile-details">
+                        <h4 style="font-size: 20px; margin: 0px 0px 0px 0px">Youtube</h4>
+                        <p style="font-size: 14px; margin-top: 0px;">8,04 M Subscribers</p>
```
Comment (umairarshadbutt) [inline index.html]: Remove inline style and manage it in the external style sheet

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @jcy2704 AND @alejandrotoledoweb 
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly work on the suggested changes.
- [ ] Remove the inline styling and use it by the external style sheet. ([Article for support](https://speckyboy.com/good-bad-css-practices/) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright))

- **Explanation:** Inline styling is a bad practice, we recommend to use CSS in external style sheet. I provide a link for your better understanding

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by jofer86 — APPROVED** [review]
## `const PROJECT_STATUS = "APPROVED"` 🚀 🚀 🚀 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- All of the required changes were met, this is approved!! you may carry on team!!
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Pull request merged**